scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1079,7 +1283,19 @@ $IotEdgeQuickstartExeTestPath = (Join-Path $QuickstartWorkingFolder "IotEdgeQuic
 $LeafDeviceExeTestPath = (Join-Path $LeafDeviceWorkingFolder "LeafDevice.exe")
 $DeploymentWorkingFilePath = Join-Path $TestWorkingFolder "deployment.json"
 
-&$InstallationScriptPath
+If ($TestName -eq "LongHaul")
```

Comment:
Make sure these parameters should not define with [ValidateNotNullOrEmpty()] attribute; since logic here will assign default value if not given.